libsolidity/interface/CompilerStack.cpp

```diff
@@ -1548,6 +1549,9 @@ string CompilerStack::createMetadata(Contract const& _contract, bool _forIR) con
 	if (m_revertStrings != RevertStrings::Default)
 		meta["settings"]["debug"]["revertStrings"] = revertStringsToString(m_revertStrings);
 
+	meta["settings"]["metadata"]["append"] =
```

Comment:
Hm?

Reply:
There's `MetadataFormat::NoMetadata` below, so perhaps this here would be way more readable as `m_metadataFormat != MetadataFormat::NoMetadata`.

Reply:
Fixed.